docs/guides/guide-delete-or-replace-metrics.md

```diff
@@ -0,0 +1,299 @@
+      "job": "vmagent",
+      "instance": "vmagent:8429"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmalert",
+      "instance": "vmalert:8880"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vminsert",
+      "instance": "vminsert:8480"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmselect",
+      "instance": "vmselect:8481"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmstorage",
+      "instance": "vmstorage-1:8482"
+    },
+    {
+      "__name__": "process_cpu_cores_available",
+      "job": "vmstorage",
+      "instance": "vmstorage-2:8482"
+    }
+  ]
+}
+
+```
+
+When you're sure with [time series selector](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors) correctness, send a POST request to [delete API](https://docs.victoriametrics.com/url-examples.html#apiv1admintsdbdelete_series) with [`match[]=<time-series-selector>`](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors) argument. For example:
+
+**Single-node VictoriaMetrics**:
+
+<div class="with-copy" markdown="1">
+
+```console
+
+curl -v http://127.0.0.1:8428/api/v1/admin/tsdb/delete_series?match[]=process_cpu_cores_available'
+```
+
+</div>
+
+**Cluster version of VictoriaMetrics**:
+<div class="with-copy" markdown="1">
+
+```console
+curl -s 'http://127.0.0.1:8481/select/0/prometheus/api/v1/series?match[]=process_cpu_cores_available'
+```
+
+</div>
+
+If operation was successful, the deleted series will stop being [queryable](https://docs.victoriametrics.com/keyConcepts.html#query-data). Storage space for the deleted time series isn't freed instantly - it is freed during subsequent [background merges of data files](https://medium.com/@valyala/how-victoriametrics-makes-instant-snapshots-for-multi-terabyte-time-series-data-e1f3fb0e0282). The background merges may never occur for data from previous months, so storage space won't be freed for historical data. In this case [forced merge](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#forced-merge) may help freeing up storage space.
+
+To trigger `force merge` on Single-node VictoriaMetrics run the following command:
+
+<div class="with-copy" markdown="1">
+
+```console
+curl -v -X POST http://127.0.0.1:8428/internal/force_merge
+```
+
+And for the VictoriaMetrics Cluster version please run:
+
+<div class="with-copy" markdown="1">
+
+```console
+curl -v -X POST http://127.0.0.1:8482/internal/force_merge
+```
+
+After the merge is complete, the data will be permanently deleted from the disk.
+
+## How to replace metrics
+
+By default, VictoriaMetrics doesn't provide a mechanism for replacing or updating data. As a workaround, take the following actions:
+
+In short, we have to do the following:
+- [export metrics to a file](https://docs.victoriametrics.com/url-examples.html#apiv1export);
```

Comment:
```suggestion
- [export time series to a file](https://docs.victoriametrics.com/url-examples.html#apiv1export);
```

Reply:
Please check the difference between metric and time series here - https://docs.victoriametrics.com/keyConcepts.html#what-is-a-metric